### PR TITLE
added v2 files

### PR DESCRIPTION
**Pull request by kenho811-b1**

### CODE REVIEW

File: create_amm_instruction_v2.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    "command": {
+        "commandType": "V2AddLiquidity",
+        "symbol": SYMBOL,
+        "baseQuantity": "1.00000000",
+        "quoteQuantity": get_adjusted_quote_amount("1.00000000", get_current_amm_price(FEE_TIER, SYMBOL),
+                                                   PRICE_LOWER_BOUND, PRICE_UPPER_BOUND),
+        "upperBound": PRICE_UPPER_BOUND,
+        "lowerBound": PRICE_LOWER_BOUND,
+        "feeTierId": FEE_TIER,
+        "tradingAccountId": TRADING_ACCOUNT_ID,
```
Comment (kenho811-b1) [inline create_amm_instruction_v2.py:81]: added TRADING_ACCOUNT_ID

File: cancel_order_v2.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "BX-TIMESTAMP": timestamp,
+    "BX-NONCE": next_nonce,
+}
+
+response = session.delete(
+    HOST_NAME
+    + "/trading-api/v1/orders"
+    + f"?orderId={ORDER_ID}"
+    + f"&symbol={SYMBOL}"
+    + f"&tradingAccountId={TRADING_ACCOUNT_ID}",
```
Comment (kenho811-b1) [inline cancel_order_v2.py:56]: added TRADING_ACCOUNT_ID

File: cancel_order_v2.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+body = {
+    "timestamp": timestamp,
+    "nonce": next_nonce,
+    "authorizer": AUTHORIZER,
+    "command": {
+        "commandType": "V2CancelOrder",
+        "orderId": ORDER_ID,
+        "handle": None,
+        "symbol": SYMBOL,
+        "tradingAccountId": TRADING_ACCOUNT_ID,
```
Comment (kenho811-b1) [inline cancel_order_v2.py:34]: added TRADING_ACCOUNT_ID

File: create_amm_instruction_v2.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+headers = {
+    "Content-type": "application/json",
+    "Authorization": f"Bearer {JWT_TOKEN}",
+    "BX-SIGNATURE": signature,
+    "BX-TIMESTAMP": timestamp,
+    "BX-NONCE": next_nonce,
+}
+
+response = session.post(
+    HOST_NAME + "/trading-api/v1/amm-instructions", json=body, headers=headers
```
Comment (kenho811-b1) [inline create_amm_instruction_v2.py:99]: the API endpoint for V2  AMM instructions is still /trading-api/v1/amm-instructions

File: create_order_v2.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+headers = {
+    "Content-type": "application/json",
+    "Authorization": f"Bearer {JWT_TOKEN}",
+    "BX-SIGNATURE": signature,
+    "BX-TIMESTAMP": timestamp,
+    "BX-NONCE": next_nonce,
+}
+
+response = session.post(
+    HOST_NAME + "/trading-api/v1/orders", json=body, headers=headers
```
Comment (kenho811-b1) [inline create_order_v2.py:54]: the API endpoint for V2  orders is still /trading-api/v1/orders

File: private_data_web_socket.py [hunk 6]
```diff
@@ -54,7 +55,8 @@ def on_open(conn):
 
 
 def open_connection():
-    ws_conn = websocket.WebSocketApp(HOST_NAME + "/trading-api/v1/private-data",
+    url = f"{HOST_NAME}/trading-api/v1/private-data?tradingAccountId={TRADING_ACCOUNT_ID}" if TRADING_ACCOUNT_ID is not None else f"{HOST_NAME}/trading-api/v1/private-data"
```
Comment (kenho811-b1) [inline private_data_web_socket.py:58]: the API endpoint for private data is still /trading-api/v1/private-data

File: read_order_v2.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+headers = {
+    "Content-type": "application/json",
+    "Authorization": f"Bearer {JWT_TOKEN}",
+}
+
+response = session.get(
+    HOST_NAME
+    + "/trading-api/v1/orders"
+    + f"?tradingAccountId={TRADING_ACCOUNT_ID}",
```
Comment (kenho811-b1) [inline read_order_v2.py:19]: added TRADING_ACCOUNT_ID

File: terminate_amm_instruction_v2.py [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+    "BX-TIMESTAMP": timestamp,
+    "BX-NONCE": next_nonce,
+}
+
+response = session.delete(
+    HOST_NAME
+    + "/trading-api/v1/amm-instructions"
+    + f"?liquidityId={LIQUIDITY_ID}"
+    + f"&symbol={SYMBOL}"
+    + f"?tradingAccountId={TRADING_ACCOUNT_ID}",
```
Comment (kenho811-b1) [inline terminate_amm_instruction_v2.py:54]: added TRADING_ACCOUNT_ID

File: terminate_amm_instruction_v2.py [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+body = {
+    "timestamp": timestamp,
+    "nonce": next_nonce,
+    "authorizer": AUTHORIZER,
+    "command": {
+        "commandType": "V2RemoveLiquidity",
+        "liquidityId": LIQUIDITY_ID,
+        "symbol": SYMBOL,
+        "tradingAccountId": TRADING_ACCOUNT_ID,
```
Comment (kenho811-b1) [inline terminate_amm_instruction_v2.py:32]: added TRADING_ACCOUNT_ID

File: README.md [hunk 10]
```diff
@@ -15,23 +15,26 @@ pip3 install -r requirements.txt
...
+More info on how to obtain them is available in [this guide](https://api.exchange.bullish.com/docs/api/rest/#overview--how-to-send-authenticated-requests).  
+
+Specifically,
+
+- [API key (Public Key, Private Key, API Metadata)](https://api.exchange.bullish.com/docs/api/rest/#overview--generate-an-api-key):
+
+- [Login Tokens (JWT token and Authorizer)](https://api.exchange.bullish.com/docs/api/rest/#overview--generate-a-jwt-token) 
 
-For a full walk-through on the usage of these scripts, start [here](https://api.exchange.bullish.com/docs/api/rest/#overview--how-to-send-authenticated-requests).  
+- Trading Account Ids
```
Comment (kenho811-b1) [inline README.md:39]: Link to be added after the section is created in the Online docs.

File: README.md [hunk 11]
```diff
@@ -15,23 +15,26 @@ pip3 install -r requirements.txt
...
-## Scripts
+More info on how to obtain them is available in [this guide](https://api.exchange.bullish.com/docs/api/rest/#overview--how-to-send-authenticated-requests).  
+
+Specifically,
+
+- [API key (Public Key, Private Key, API Metadata)](https://api.exchange.bullish.com/docs/api/rest/#overview--generate-an-api-key):
+
+- [Login Tokens (JWT token and Authorizer)](https://api.exchange.bullish.com/docs/api/rest/#overview--generate-a-jwt-token) 
 
-For a full walk-through on the usage of these scripts, start [here](https://api.exchange.bullish.com/docs/api/rest/#overview--how-to-send-authenticated-requests).
```
Comment (kenho811-b1) [inline README.md:32]: I moved and reworded it here

https://github.com/bullish-exchange/api-examples/pull/18/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R31

**Review by dexterhuang-blockone — APPROVED** [review]

**Pull request merged**